README.md

```diff
@@ -182,6 +182,15 @@ Refer to the [Komodo Developer Docs](https://developers.komodoplatform.com/basic
 - [Git flow and general workflow](./docs/GIT_FLOW_AND_WORKING_PROCESS.md)
 - [Komodo Developer Docs](https://developers.komodoplatform.com/basic-docs/atomicdex/introduction-to-atomicdex.html)
 
+## 🔄 About Atomic Swaps & HTLC
+
+Atomic swaps in Komodo rely on Hash Time-Locked Contracts (HTLC), which ensure that cross-chain trades either execute completely or not at all.
+
+If you're new to HTLCs, here’s a minimal working example implemented in Solidity:
+👉 [Solidity HTLC PoC](https://github.com/Jango-ai/atomic-swap-workshop)
+
+This is useful for understanding the underlying mechanics behind Komodo’s swap engine.
+
```

Comment:
We already have a link to HTLCs here https://github.com/KomodoPlatform/komodo-defi-framework/blob/7ef08e47667c9cf20b900733619107d47ef0aa1f/README.md#L51
Why do you think this additional explanation is needed?